microsoft_outlook_tenant_id/README.rst

```diff
@@ -0,0 +1,88 @@
+===============================
```

Comment:
please use readme fragments

Reply:
I'd rename the module to `microsoft_outlook_single_tenant`

Reply:
Thanks `microsoft_outlook_single_tenant` is a good name indeed